examples/argocd-manager-rolebinding.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: authorization.openshift.io/v1
+kind: RoleBinding
+metadata:
+  name: argocd-manager-rolebinding
```

Comment:
Should you specify the namespace as well in the metadata?

Reply:
I wanted to keep it generic

Reply:
I'd just make it something like `namespace: <application_namespace>`